ibridges/interactive.py

```diff
@@ -0,0 +1,47 @@
+"""Interactive authentication with iRODS server.
+"""
+
+import json
+import os
+from pathlib import Path
+from typing import Optional, Union
+from getpass import getpass
+
+from ibridges.irodsconnector.session import Session
+
+def interactive_auth(password: Optional[str] = None, irods_env_path: Optional[Union[str, Path]] =
+                 os.path.expanduser("~/.irods/irods_environment.json")) -> Session:
```

Comment:
Perhaps it would be good to have a DEFAULT_IENV_PATH or something of the like? I could see it being reused at some point.

Reply:
Where shall we then define this global variable? In the session.py?